v1/consumer/consumer.gunk

```diff
@@ -0,0 +1,393 @@
+	PerMinuteCallLimit string `pb:"2" json:"per_minute_call_limit"`
+	PerHourCallLimit   string `pb:"3" json:"per_hour_call_limit"`
+	PerDayCallLimit    string `pb:"4" json:"per_day_call_limit"`
+	PerWeekCallLimit   string `pb:"5" json:"per_week_call_limit"`
+	PerMonthCallLimit  string `pb:"6" json:"per_month_call_limit"`
+}
+
+type UpdateConsumerRedirectURLRequest struct {
+	ConsumerID  string `pb:"1" json:"consumer_id"`
+	RedirectURL string `pb:"2" json:"redirect_url"`
+}
+
+type UpdateConsumerRedirectURLResponse struct {
+	Consumer Consumer `pb:"1" json:"consumer"`
+}
+
+// ConsumerService provices consumer operations.
+type ConsumerService interface {
+	// EnableOrDisableConsumer
+	//
+	// +gunk http.Match{
+	//         Method: "PUT",
+	//         Path:   "/v1/management/consumers/{ConsumerID}",
+	//         Body:   "*",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"Consumer"},
+	//         Description: "Set enable or disable consumer",
+	//         Summary:     "Set consumer enable or disable.",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
+	//                         Schema: openapiv2.Schema{
+	//                                 JSONSchema: openapiv2.JSONSchema{
+	//                                         Ref: "#/definitions/consumerEnableOrDisableConsumerResponse",
+	//                                 },
+	//                         },
+	//                 },
+	//                 "404": openapiv2.Response{
+	//                         Description: "Returned when the resource is not found.",
+	//                 },
+	//         },
+	//         Security: []openapiv2.SecurityRequirement{
+	//                 {
+	//                         SecurityRequirement: map[string]openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                 "OAuth2": openapiv2.SecurityRequirement_SecurityRequirementValue{
+	//                                         Scope: []string{
+	//                                                 "write",
+	//                                         },
+	//                                 },
+	//                         },
+	//                 },
+	//         },
+	// }
+	EnableOrDisableConsumer(EnableOrDisableConsumerRequest) EnableOrDisableConsumerResponse
+
+	// GetConsumerCallLimits
+	//
+	// +gunk http.Match{
+	//         Method: "GET",
+	//         Path:   "/v1/management/consumers/{ConsumerID}/consumer/call-limits",
+	// }
+	// +gunk openapiv2.Operation{
+	//         Tags:        []string{"Consumer"},
+	//         Description: "Returns a list up to 20 consumer call limits data.",
+	//         Summary:     "Get consumer call lists",
+	//         Responses: map[string]openapiv2.Response{
+	//                 "200": openapiv2.Response{
+	//                         Description: "Request executed successfully.",
```

Comment:
missing schema tag here. 

``` Schema: openapiv2.Schema{
	//                                 JSONSchema: openapiv2.JSONSchema{
	//                                         Ref: "#/definitions/consumerGetConsumerCallLimitsResponse",
	//                                 },
	//                         }
```

Reply:
do we need to use schema tag for get request?